main.tf

```diff
@@ -457,9 +465,15 @@ resource "aws_ecs_service" "ignore_changes_task_definition" {
     }
   }
 
+  triggers = local.redeployment_trigger
+
   lifecycle {
     ignore_changes = [task_definition]
   }
+
+  # Avoid race condition on destroy.
+  # See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service
+  depends_on = [aws_iam_role.ecs_service, aws_iam_role_policy.ecs_service]
```

Comment:
should we add this to our redis/kvstore module as well?

Reply:
Yes, but I have something different in mind for this modules.  I will create a ticket for it.